### ISSUE TITLE
Not all NFTs checked out (mainnet-beta)

### ISSUE THREAD

**Issue by annabnn812** [metaplex-foundation/metaplex#1516]
Hey guys! Can someone help me? Not all NFTs checked out (mainnet-beta) I am used Candy Machine-v2 . Uploaded (545) out of (555). How to fix it without deleting the folder " .cache" ?  I know , I know about the instruction "  In this case, you will need to re-run the upload command. Only after the verify command succeeds should you make your Candy Machine live." ---- THIS DOES NOT WORK FOR ME! I see some error , its looks like " Size 0 undefined
upload was not successful, please re-run. TypeError: Cannot read properties of undefined (reading 'index')" But I didnt changed index or something else.

![image](https://user-images.githubusercontent.com/86491435/148879770-5bacaec7-5fe3-4110-b306-78ad33ce471a.png)

![image](https://user-images.githubusercontent.com/86491435/148879810-822857d1-cf1f-4a4b-a7bd-92a49cd23bc6.png)

![image](https://user-images.githubusercontent.com/86491435/148879843-4b12ecc2-b6dd-4690-a78e-c58e61b60217.png)

![image](https://user-images.githubusercontent.com/86491435/148882888-40c30771-23f6-4b19-849f-24c08ba347c4.png)

**Comment by neurofoo** [comment]
I've had a similar issue. Create a v2 candy machine. All good except one index has this error. I've tried running the upload command again as instructed in the docs. Nothing. 

Are there more docs/SOP to figure out how to understand why the index fails?

**Comment by neurofoo** [comment]
If anyone stumbles upon this, there seems to be an issue in the verification code. It doesn't correctly skip over files that have already been verified.

Go to your cache file and remove all instances of 

```
"verifyRun": true
```

This will force the code to recheck all files again.

Then rerun the verify_upload code.

**Comment by stegaBOB** [comment]
As for @neurofoo, I think your issue is the same as #1856. And for @annabnn812, it's hard to help solve your issue since its based on a CLI that has changed so much in the past couple months. If you haven't solved your issue by now, please feel free to open up a new issue with our new issue template form.

**Comment by psychofaced** [comment]
what if i do change the "verifyRun": false into "verifyRun": true manually ? 
I did that and re-run the verify_upload command and it say "ready to deploy!" . Does it will causing the problem on the minting ?
